### DISCUSSION TITLE
SQL Server Management Studio

### DISCUSSION THREAD

**Discussion by eoliverdev** [balta-io/forum#283] [Tira Dúvidas] [answered]
Olá Balta, estou fazendo o módulo 'Fundamentos do SQL Server'. Eu tenho dificuldades de instalar o Docker porque uso o computador da empresa ( é meio burocratico, mas estou correndo atrás de instalar) mas no momento eu não consigo instalar o Docker. Mas eu tenho o SQL Server Management Studio 18 instalado e consegui intalar o Azure Data Studio, porque ele tem a versão usuário que não necessita usuário administrador. Parece que você comenta no início do módulo que da pra usar o SSMS mas não fala como. Dá pra seguir dessa forma sem o Docker? Se sim, pode me orientar a respeito por favor? Grato, Emerson.

**Comment by andrebaltieri** [comment, marked as answer]
Boa noite, @eoliverdev, como vai?
Opa, dá sim, 100%! Só precisa ter os dados do servidor como IP, porta, usuário e senha, daí só fazer a conexão por dentro do SSMS ou Azure Data Studio.

Lembrando que o usuário precisa ter permissão para criar banco de dados.

**Comment by eoliverdev** [comment]
Consegui instalar o SQL Server!

Só que, como você já tinha alertado, na hora que fui criar o banco 'Curso' deu **_"Permissão CREATE DATABASE negada no banco de dados 'master'"_**. Preciso direcionar o suporte da minha empresa pra me dar essa autorização, por favor pode me ensinar como consigo essa permissão, o que tenho que fazer pra mudar isso?

**Reply by andrebaltieri** [reply]
Boa tarde, @eoliverdev, como vai?
Você precisa de um usuário com permissão para ciração do banco, eles vão saber como criar isto internamente no seu usuário do AD.

Se conseguir um usuário com perfil ADMIN pelo menos para o SQL Server local, melhor ainda :)

**Comment by MarcioFreitasUFSC21** [comment]
boa tarde, professor! também estou com dificuldades de instalar o Docker aqui no lab de onde trabalho, mas já tenho isntalado o mySqL workbench.com isso, eu poderia usá-lo para conectar à minha API que estou construindo a partir de suas orientações na formação backend com node.js?

**Reply by andrebaltieri** [reply]
Boa tarde, @MarcioFreitasUFSC21, como vai?

Poderia sim, porém fica diferente do curso, minha recomendação é instalar o SQL Server, mas se não der, pode tentar com o MySQL por conta.

Caso esteja usando o Windows, você não precisa do Docker se não quiser. Pode instalar o SQL Server Express:
https://www.microsoft.com/pt-br/download/details.aspx?id=101064

**Reply by MarcioFreitasUFSC21** [reply]
ok, professor!obrigado então!

**Reply by paulinhosimao** [reply]
Boa noite professor, @andrebaltieri , consigo fazer a importação da sua base sem instalar o sql no docker? Eu já tenho o banco sql server local instalado, por ser maquina da empresa, não libera instalar o docker.  Ou tem os registros em algum arquivo para download, aí vou aplicando os inserts.

**Reply by andrebaltieri** [reply]
Bom dia @paulinhosimao, como vai?
Consegue sim, sem problemas... Tendo o SQL instalado, é só conectar nele com Azure Data Studio (Igual mostro na aula -- ou usar outro programa que você goste, como SSMS) e fazer o restore.

**Reply by paulinhosimao** [reply]
> Bom Dia@paulinhosimao, como vai? Consegue sim, sem problemas... Tendo o SQL instalado, é só conectar nele com Azure Data Studio (Igual mostro na aula -- ou usar outro programa que você goste, como SSMS) e fazer o restore.

Bom dia professor, ótimo vou tentar fazer então, eu tenho o SSMS instalado.

**Reply by paulinhosimao** [reply]
> Bom Dia@paulinhosimao, como vai? Consegue sim, sem problemas... Tendo o SQL instalado, é só conectar nele com Azure Data Studio (Igual mostro na aula -- ou usar outro programa que você goste, como SSMS) e fazer o restore.

Professor @andrebaltieri então pelo SSMS faço a conexão local normal, e tem a opção de importar o arquivo .bacpac?

**Reply by andrebaltieri** [reply]
@paulinhosimao tem sim!

**Reply by MarcioFreitasUFSC21** [reply]
> Bom dia @paulinhosimao, como vai? Consegue sim, sem problemas... Tendo o SQL instalado, é só conectar nele com Azure Data Studio (Igual mostro na aula -- ou usar outro programa que você goste, como SSMS) e fazer o restore.

professor, boa tarde! por favor, poderia dizer em qual aula o senhor mostra essa configuração do SQL server com o Azure Data Studio e o SSMS, pois já os tenho instalados aqui na minha máquina?
desde já, agradeço-lhe a atenção e fico no aguardo.
Att.

**Reply by paulinhosimao** [reply]
> Bom Dia@paulinhosimao, como vai? Consegue sim, sem problemas... Tendo o SQL instalado, é só conectar nele com Azure Data Studio (Igual mostro na aula -- ou usar outro programa que você goste, como SSMS) e fazer o restore.

Deu certo professor @andrebaltieri , muito obrigado.

**Comment by williamdev93** [comment]
Eu uso o sql do próprio visual studio, e da super certo também.
